assessments/projects/karmada/self-assessment.md

```diff
@@ -0,0 +1,430 @@
+# Self-assessment for Karmada
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+  * [Threat Modeling with STRIDE](#threat-modeling-with-stride)
+
+## Metadata
+
+A table at the top for quick reference information, later used for indexing.
+
+|   |  |
+| -- | -- |
+| **Software** | [A link to Karmada’s repository.](https://github.com/karmada-io/karmada)  |
+| **Security Provider** | No. The primary function of the project is to run Kubernetes clusters across clouds. Security is not the primary objective. |
+| **Languages** |<ul><li>Go</li><li>Shell</li><li>Smarty</li><li>Mustache</li><li>Makefile</li><li>Dockerfile</li></ul> |
+| **SBOM** | SBOM generated using **FOSSA-cli** tool on the latest code base. [Link to SBOM](https://github.com/Rana-KV/tag-security/blob/main/assessments/projects/karmada/docs/Karmada_SBOM_cyclonedx.bom.json) |
```

Comment:
This seems to be your personal repo. Please provide the link to the SBOM generated/released by the team. If one isn't available, kindly mark as not available

Reply:
Thanks for your review @ragashreeshekar.

Currently the team doesn't have any public SBOM available. I will record it as "Not Available".

Reply:
Is there any example of public SBOM?
The team is now running Fossa against each commit and the report is publicly available. Such as the latest report can be found at [here](https://app.fossa.com/projects/custom%2B28176%2Fgithub.com%2Fkarmada-io%2Fkarmada/refs/branch/master/65376b28d5037c27ff7ec0e56542c2a345d1a120/preview)

Reply:
@RainbowMango 

Sorry, my bad. I think the link you shared it not restricted. I will update the document again.